### ISSUE TITLE
Petition JSON is...unexpected

### ISSUE THREAD

**Issue by yoni** [jm/we_the_people#11]
It's a bit unexpected that calling `petition.to_json` returns an object which has the attributes as a nested value.

For example, I'd expect `petition.to_json` to return:

```
{
  "id":"1234",
  "title":"xyz",
   ...
}
```

but instead I get:

```
{
  "parent":null,
  "attributes": {
    "id":"1234",
    "title":"xyz",
    ...
  }
}
```

**Comment by jm** [comment]
Yeah that's a product of implementation.  It'll be easy to switch it up (i.e. simply have `to_json` return `attributes`)

**Comment by jm** [comment]
Fixed in 9a1efbd!

**Comment by yoni** [comment]
Thanks @jm!
